### PR TITLE
Bump version to 2.21.3

### PR DESCRIPTION
**Pull request by matthewfala**
Pulls latest amazonlinux as base image that includes a patch for
HIGH CVE-2021-43527.

Signed-off-by: Matthew Fala <falamatt@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by hossain-rayhan — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+* Amazon Kinesis Streams for Fluent Bit 1.8.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Same as `2.21.2`, this release includes the following fixes for AWS customers that we are working on getting accepted upstream:
+* Bug - Fix return value from `tls_net_read` [fluentbit:4098](https://github.com/fluent/fluent-bit/issues/4098)
+* Bug - Downgrade `mbedtls` to 2.24.0 to fix the performance regression issue in `mbedtls` 2.26.0 [fluentbit:4110](https://github.com/fluent/fluent-bit/issues/4110)
+* Bug - Resolve IMDSv1 fallback error introduced in 2.21.0 [aws-for-fluent-bit:259](https://github.com/aws/aws-for-fluent-bit/issues/259)
+
+Important Note:
+* A security vulnerability was found in [amazonlinux](https://access.redhat.com/security/cve/CVE-2021-43527) which we use as base image to our `aws-for-fluent-bit` image. This new image will be based on an updated version of amazonlinux that resolves this CVE. It is highly recommended to upgrade your existing workload or run new workload with this latest version (2.21.3) of `aws-for-fluent-bit` image.
```
Comment (PettitWesley) [inline CHANGELOG.md:17]: I thought we agreed we wouldn't put the "highly recommended" warning? And let customers decide how scared to be.

**Review by PettitWesley — APPROVED** [review]

File: AWS_FOR_FLUENT_BIT_STABLE_VERSION [hunk 2]
```diff
@@ -1 +1 @@
-2.21.2
\ No newline at end of file
+2.21.3
```
Comment (PettitWesley) [inline AWS_FOR_FLUENT_BIT_STABLE_VERSION:1]: sorry I missed this
we can;t bump stalbe until the new release is out
this will cause the automation to fial

**Review by PettitWesley — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (matthewfala) [reply CHANGELOG.md:17]: The forced push looks like it didn't make it through.

Comment (matthewfala) [reply CHANGELOG.md:17]: I edited this locally but the forced push looks like it didn't make it through.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+* An Amazon Linux 2 Base
+* Fluent Bit [1.8.9](https://fluentbit.io/announcements/v1.8.9/)
+* Amazon CloudWatch Logs for Fluent Bit 1.6.4
+* Amazon Kinesis Streams for Fluent Bit 1.8.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Same as `2.21.2`, this release includes the following fixes for AWS customers that we are working on getting accepted upstream:
+* Bug - Fix return value from `tls_net_read` [fluentbit:4098](https://github.com/fluent/fluent-bit/issues/4098)
+* Bug - Downgrade `mbedtls` to 2.24.0 to fix the performance regression issue in `mbedtls` 2.26.0 [fluentbit:4110](https://github.com/fluent/fluent-bit/issues/4110)
+* Bug - Resolve IMDSv1 fallback error introduced in 2.21.0 [aws-for-fluent-bit:259](https://github.com/aws/aws-for-fluent-bit/issues/259)
```
Comment (zhonghui12) [inline CHANGELOG.md:14]: I have a question, was your fix for this one merged into Fluent Bit?

**Review by PettitWesley — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (matthewfala) [reply CHANGELOG.md:14]: No, the PRs are all still open.
https://github.com/fluent/fluent-bit/pull/4100
https://github.com/fluent/fluent-bit/issues/4110
https://github.com/fluent/fluent-bit/pull/4184

**Pull request merged**